src/i18n/en.json

```diff
@@ -135,16 +161,37 @@
     "title": "My information",
     "visibility": "The following information about you is stored in Helsinki profile.",
     "doGoToAccountManagement": "Go to account management",
-    "authenticationMethod": "Authentication method"
+    "authenticationMethod": "Authentication method",
+    "primaryAddress": "This is your primary address",
+    "permanentAddress": "EN: Vakinainen osoite",
+    "temporaryAddress": "EN: Väliaikainen osoite",
+    "permanentForeignAddress": "EN: Väliaikainen ulkomainen osoite",
+    "primaryEmail": "This is your primary email",
+    "primaryPhone": "This is your primary phone number",
+    "verifiedDataInformationPrefix": "EN: Merkityt",
+    "verifiedDataInformationAriaPrefix": "EN: Alla olevat",
+    "verifiedDataInformation": "EN: <0></0> tiedot on vahvistettu virallisesta väestötietojärjestelmästä, eikä niitä ei voi tämän takia muokata Helsinki Profiilissa. Voit tutustua halutessasi väestötietojärjestelmään tallennettuihin tietoihisi suomi.fi verkkopalvelussa: <1>siirry suomi.fi palveluun </1>",
+    "verifiedDataInformationLink": "https://testipalvelu.apro.tunnistus.fi",
```

Comment:
Yes, there is no way to make environment dependant translations at the moment. It should be done when there is actual need. Maybe all links can just point to production versions until other versions are actually required. This url was given to me, because suomi.fi was apparently wrong.